deploy/jenkins-ci/Makefile.operator.jenkins

```diff
@@ -91,25 +91,24 @@ operator-build-release:
 	OPERATOR_QUAY_NAME="$(QUAY_OPERATOR_NAME)" \
 	  $(MAKE) clean build
 
+# prepare operator tags
+QUAY_OPERATOR_TAG = $(QUAY_OPERATOR_NAME):$(VERSION_TAG)
+ifneq ($(IS_SNAPSHOT),y)
+   # Create or update vX.Y tags (without the patch number)
+   # quay.io/kiali-operator
+   QUAY_OPERATOR_TAG := $(QUAY_OPERATOR_TAG) $(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)
+endif
+
 operator-push-quay:
 ifdef QUAY_USER
 ifdef QUAY_PASSWORD
 	@echo "Logging in to Quay.io..."
 	@docker login -u "$(QUAY_USER)" -p "$(QUAY_PASSWORD)" quay.io
 endif
 endif
-	OPERATOR_CONTAINER_VERSION="$(VERSION_TAG)" \
+	QUAY_OPERATOR_TAG="$(QUAY_OPERATOR_TAG)" \
 	  OPERATOR_QUAY_NAME="$(QUAY_OPERATOR_NAME)" \
-	 $(MAKE) push
-ifneq ($(IS_SNAPSHOT),y)
-   # Create or update vX.Y tags (without the patch number)
-   # quay.io/kiali-operator
-	docker tag "$(QUAY_OPERATOR_NAME):$(VERSION_TAG)" "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-	docker push "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-	docker rmi "$(QUAY_OPERATOR_NAME):v$(OPERATOR_VERSION_BRANCH)"
-endif
-   # Clean-up
-	docker rmi "$(QUAY_OPERATOR_NAME):$(VERSION_TAG)"
+	 $(MAKE) -e DOCKER_CLI_EXPERIMENTAL=enabled container-multi-arch-push-kiali-operator-quay
```

Comment:
This fails.

This command is running under the kiali-operator repository context. Thus, it's using this Makefile: https://github.com/kiali/kiali-operator/blob/master/Makefile.

I think you will need to move the changes on this PR to the operator repository: https://github.com/kiali/kiali/pull/3462/files

Reply:
But this PR deps https://github.com/kiali/kiali/blob/master/make/Makefile.container.mk#L66-L120
not sure this is a good idea to maintain this in two repo.

Reply:
We might have to just copy-n-paste these [Makefile.container.mk targets](https://github.com/kiali/kiali/blob/master/make/Makefile.container.mk#L66-L126) from the kiali/kiali repo directly into the [kiali/kiali-operator Makefile](https://github.com/kiali/kiali-operator/blob/master/Makefile):

Reply:
@israel-hdez @jmazzitelli 

PR created https://github.com/kiali/kiali-operator/pull/216

copy but from this PR. here some fix

download url fix: https://github.com/kiali/kiali/pull/3473/commits/52d01a0b5bf7aa7b9d21500ed92801c522f3a974 
upgraded `moby/buildkit` to v0.8.0：https://github.com/kiali/kiali/pull/3473/commits/0cece15f967720ee443dbf790fc64cfb81c20205

Reply:
It's working now, but not as expected.
Issued command is: `docker buildx build --push --pull --no-cache --builder=kiali-builder  --tag=quay.io/****/kiali-operator:v1.28.0 -f kiali/kiali-operator/build/Dockerfile kiali/kiali-operator`

Notice there aren't any `--platform` flags. So, in my test run, I got a non multi-arch image: https://quay.io/repository/edgarhz/kiali-operator?tab=tags

Also, notice the only pushed tag is `v1.28.0`, the `v1.28` tag is missing.

I see some mistakes:
* Here you are using `QUAY_OPERATOR_TAG` environment variable, while in the operator repo, you are using `OPERATOR_QUAY_TAG`.
* In the operator PR, you are using `TARGET_ARCHS` environment variable which is not declared under that context.